src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipServiceTest.java

```diff
@@ -96,72 +91,37 @@ void givenDto_whenCreateLP_thenLPCreatedWithSubmissionIdAndTransactionUpdated()
         when(repository.insert(limitedPartnershipDao)).thenReturn(limitedPartnershipDao);
 
         Transaction transaction = buildTransaction();
-
         // when
         String submissionId = service.createLimitedPartnership(transaction, limitedPartnershipDto, REQUEST_ID, USER_ID);
 
         // then
+        verify(transactionService).hasExistingLimitedPartnership(transactionApiCaptor.capture());
```

Comment:
the transactionApiCaptor isn't used so can just replace with transaction on line 98 and 102
for 102 you can replace some of the parameters with the real values like limitedPartnershipDto and use any(<classname>) instead of any.

Reply:
Removed the captor

Reply:
Removed the captor